plugins/cpp/test/src/cppparsertest.cpp

```diff
@@ -526,10 +538,15 @@ TEST_F(CppParserTest, Enum)
           break;
       }
     }
+  });
+}
 
-    model::CppFunction fieldFunction = _db->query_value<model::CppFunction>(
+TEST_F(CppParserTest, Fields)
```

Comment:
Some tests cases were put in the `CppParserTest::Enums` test case, but they were really checking class fields. So I created this new  `CppParserTest::Fields` test case for them.